src/app/pages/home/rolodex/rolodex.page.html

```diff
@@ -1,43 +1,6 @@
-<ion-toolbar>
-   <ion-searchbar animated="false" placeholder="Search cards" [(ngModel)]="searchTerm"></ion-searchbar>
-</ion-toolbar>
-
 <ion-content class="card-background-page">
-  <ion-slides pager="false" [options]="slideOpts" class="rolodex-container">
-    <ion-slide button [routerLink]="['/user', user.id]" *ngFor="let user of (users | async | filter:searchTerm)" lines="none">
-      <ion-item>
-        <ion-card>
-            <ion-item lines="none">
-            <ion-avatar slot="start">
-              <img src="./assets/imgs/Beach.jpg">
-            </ion-avatar>
-                <ion-card-subtitle>
-                  <h1>{{ user.name }}</h1>
-                    Mutant Consultant
-                    <br>
-                    {{ user.email }} | (313) 777-9311
-                    <br>
-                    www.Xmen.com
-                </ion-card-subtitle>
-            </ion-item>
-          <div class="icon-format">
-          <ion-icon name="logo-instagram" size="large"></ion-icon>
-          <ion-icon name="logo-facebook" size="large"></ion-icon>
-          <ion-icon name="logo-linkedin" size="large"></ion-icon>
-          </div>
-
-          <ion-card-content class="card-subtitle">
-            <p style="text-align:left">You choose to believe what you believe in because your soul when it’s unwhole is uneven.</p>
-              <p style="color:#0C62BB;">#RealityFromBelief #BeliefFromReality.</p>
-          </ion-card-content>
-
-        </ion-card>
-      </ion-item>
-    </ion-slide>
-
 
-
-    <ion-item *ngFor="let card of cards | async" lines="none">
```

Comment:
Why take out the "async"? Don't we need it to dynamically add cards to the stack and display in the rolodex?

Reply:
Read the `profile.page.ts` file to see how this is used. Since we needed to do something when there are no cards in the list, we need to actually read the size of the list. This is impossible with the Observable object so we subscribe to the object and either display the message encouraging the user to create a new card or display the list. It will still be dynamically populated (cards are added and removed as soon as the Firebase changes).

We technically don't use that two-phase process on the Rolodex page but it was a relatively simple setup and lends itself to extension in the future that an Observable would prevent.